03-Calling-an-API/src/auth/authService.js

```diff
@@ -0,0 +1,141 @@
+import auth0 from "auth0-js";
+import { EventEmitter } from "events";
+import { AUTH_CONFIG } from "./auth0-variables";
+
+const webAuth = new auth0.WebAuth({
+  domain: AUTH_CONFIG.domain,
+  redirectUri: AUTH_CONFIG.callbackUrl,
+  clientID: AUTH_CONFIG.clientId,
+  responseType: "token id_token",
```

Comment:
Sample 03 is designed to demonstrate how to call an API using an Access Token, whereas Sample 01 only needs the ID token. In the tutorial, the reader is asked to modify the response type to include `token` so that they understand the working parts needed in order to get this functionality to work.